services/analyzer/app.py

```
@@ -195,10 +253,53 @@ async def analyze_prompt(
-    Analyze a prompt for security threats.
-    Uses both heuristic rules and ML-based detection.
-    """
+    """Analyze a prompt for security threats.
+
+        Authenticates the caller via API key, runs both heuristic and ML-based
+        detection on the submitted prompt, logs an audit record, and returns a
+        structured risk assessment.
+
+        Args:
+            request (AnalysisRequest): The analysis request containing:
+                - ``prompt`` (str): The text to analyze (1–10,000 characters).
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)